src/blueapi/service/main.py

```diff
@@ -312,6 +312,8 @@ def set_state(
         and new_state in _ALLOWED_TRANSITIONS[current_state]
     ):
         if new_state == WorkerState.PAUSED:
+            if state_change_request.defer is None:
+                state_change_request.defer = False
```

Comment:
Or can make StateChangeRequest.defer `bool` not `bool | None`- it's only going to be `None` if it's explicitly set to be but I don't see the use case for doing so?